dask/blockwise.py

```diff
@@ -426,6 +484,15 @@ def __dask_distributed_unpack__(cls, state, dsk, dependencies):
             for ind, is_list in zip(state["indices"], state["is_list"])
         ]
 
+        # Unpack io_deps state
+        io_deps = state["io_deps"].copy()
```

Comment:
```suggestion
        io_deps = {}
```
Is this `copy` necessary, or could we start from an empty dict?